tests/functional/aws-node-sdk/test/multipleBackend/objectCopy/objectCopy.js

```diff
@@ -292,6 +304,52 @@ function testSuite() {
             });
         });
 
+        it('should copy an object on AWS with encryption', done => {
```

Comment:
please be more specific about what type of encryption -- aws server side encryption